gobblin-runtime/src/main/java/org/apache/gobblin/runtime/HiveRegTaskStateCollectorServiceHandlerImpl.java

```diff
@@ -38,17 +37,19 @@
 @Slf4j
 public class HiveRegTaskStateCollectorServiceHandlerImpl implements TaskStateCollectorServiceHandler {
 
+  private static final String HIVE_REG_PUBLISHER_CLASS = "hive.reg.publisher.class";
```

Comment:
This naming is ambiguous, better to equip it with some information about task-collection-service